code/modules/unit_tests/outfit_sanity.dm

```diff
@@ -29,10 +29,13 @@
 		CHECK_OUTFIT_SLOT(ears, ITEM_SLOT_EARS)
 		CHECK_OUTFIT_SLOT(glasses, ITEM_SLOT_EYES)
 		CHECK_OUTFIT_SLOT(id, ITEM_SLOT_ID)
-		CHECK_OUTFIT_SLOT(suit_store, ITEM_SLOT_SUITSTORE)
 		CHECK_OUTFIT_SLOT(l_pocket, ITEM_SLOT_LPOCKET)
 		CHECK_OUTFIT_SLOT(r_pocket, ITEM_SLOT_RPOCKET)
-
+		if (ispath(outfit.back, /obj/item/mod/control))
```

Comment:
Run this one by me again?

Reply:
mod units need to extend, as suit storage slots only work with a suit, and mod control units are not a suit but a back item. this means that outfits that have internals on (like chief engi's mod outfit) fail checks

Reply:
I see, I'm not sure a better way to do this right now.

Can you add a comment to the test explaining this?

Reply:
sure